Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -148,14 +147,20 @@ bool ScrollAnchoringController::didFindPriorityCandidate(Document& document)
             }
             iterElement = iterElement->parentElement();
         }
-        return candidateElement;
+        if (!iterElement)
+            return nullptr;
+        auto candidateResult = examineCandidate(*candidateElement);
```

Comment:
`examineCandidate` is not a great function name.

Reply:
Changed this to examineAnchorCandidate.